test/plugins/reqIdHeaders.test.js

```diff
@@ -94,27 +94,28 @@ describe('request id headers', function() {
         });
     });
 
-    it('GH-1086: should use default uuid request id if none provided', function(
-        done
-    ) {
-        SERVER.get('/1', function(req, res, next) {
-            assert.ok(req.getId());
-            assert.ok(validator.isUUID(req.getId()));
-            res.send('hello world');
-            return next();
-        });
+    it(
```

Comment:
This breaking of `it` forces an extra indent. Anyway to change it so it doesn't?

Reply:
Unfortunately no option yet :(